app/views/posts/show.html.erb

```diff
@@ -1 +1,15 @@
-<h1>All posts show</h1>
\ No newline at end of file
+<!-- app/views/posts/show.html.erb -->
+<div class="card">
+  <h1><%= @post.title %></h1>
```

Comment:
👍 Great progress on incorporating comments into your project! To align with the wireframe specifications, consider the following enhancement:

- [ ] **Displaying Counts Alongside Post Title:** According to the wireframe, the UI should showcase not only the post title but also the counts for comments and likes alongside it. Ensure that the comments count and like count are visibly presented next to the post title, providing users with a quick overview of the post's engagement metrics.

| Your | Design |
| --- | --- |
|  ![image](https://github.com/NoelLincoln/blog-app/assets/55840999/1fb50bc5-3a74-4c54-98fc-eae18d1ae089)| ![image](https://github.com/NoelLincoln/blog-app/assets/55840999/865a6d0c-7824-4ccd-9744-9672493a2ea8)|

By making this adjustment, you'll adhere more closely to the wireframe design, offering a more intuitive and informative user interface. Keep up the good work! 🚀👏